cmdutil/constant.go

```diff
@@ -15,65 +27,120 @@ type ThemeType struct {
+	EndScript:    "",
+	BeforeScript: "https://hosted-pages.lrinternal.com/Themes/Theme-Default/profile/js/profile-before-script.js",
```

Comment:
Please check the domain for this theme, it's still - https://hosted-pages.lrinternal.com